my-site.rb

```diff
@@ -2,7 +2,19 @@
 
 class MySite < Sinatra::Base
 
-  get "/" do
```

Comment:
Have a default "root route" is still good to have even if you want to have an explicit index.html page